linux_os/guide/system/accounts/accounts-session/accounts_polyinstantiated_tmp/oval/shared.xml

```diff
@@ -37,8 +37,8 @@
   </ind:textfilecontent54_test>
 
   <ind:textfilecontent54_object id="obj_tmp_in_namespace_conf" version="1">
-    <ind:filepath>/etc/security/namespace.conf</ind:filepath>
-    <ind:pattern operation="pattern match">^\s*/tmp\s+/tmp/tmp-inst/\s+level\s+root,adm$</ind:pattern>
+    <ind:filepath operation="pattern match">^/etc/security/(?:namespace\.conf|namespace\.d/.*conf)$</ind:filepath>
+    <ind:pattern operation="pattern match">^\s*/tmp\s+/tmp/tmp-inst/\s+\S+\s+\S+\s*$</ind:pattern>
```

Comment:
Can you please explain this regex change? This allows arbitrary arguments after first two...

Reply:
My first way to explain is in commit: https://github.com/ComplianceAsCode/content/pull/8609/commits/aa3afa52f43129197171290eb52fe54811bda9c4